src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -24,7 +25,11 @@ public void DumpRuntimeInformationToConsole()
             string lcr = PlatformDetection.LibcRelease;
             string lcv = PlatformDetection.LibcVersion;
 
-            Console.WriteLine($@"{dvs} OS={osd} OSVer={osv} OSArch={osa} Arch={pra} Framework={frd} LibcRelease={lcr} LibcVersion={lcv}");
+            Console.WriteLine($@"### CONFIGURATION: {dvs} OS={osd} OSVer={osv} OSArch={osa} Arch={pra} Framework={frd} LibcRelease={lcr} LibcVersion={lcv}");
+
+            Console.WriteLine($"### BINARIES: {Path.GetDirectoryName(typeof(object).Assembly.Location)} (drive {new DriveInfo(Path.GetDirectoryName(typeof(object).Assembly.Location)).DriveFormat})");
+            Console.WriteLine($"### TEMP PATH: {Path.GetTempPath()} (drive {new DriveInfo(Path.GetTempPath()).DriveFormat})");
+            Console.WriteLine($"### CURRENT DIRECTORY: {Environment.CurrentDirectory}");
```

Comment:
Looks like this is going to conflict with https://github.com/dotnet/corefx/pull/29816/files#diff-2096cc325fd0971f6ab660d3053eddf9R30

Reply:
@danmosemsft maybe I can revert this change here and you could pickup this changes in your other PR?

Reply:
Sure